python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -76,6 +77,17 @@ class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
         doc="Weight exposures by airmass? Useful if there are relatively few high-airmass observations.",
         default=True,
     )
+    modelWidth = pexConfig.Field(
+        dtype=int,
+        doc="Width of the region around detected sources to include in the DcrModel."
+            "Set to a negative number to disable, which will include all pixels.",
+        default=3,
+    )
+    useSubfilterMidpoint = pexConfig.Field(
```

Comment:
Could this be called `useMidpoint` like in ip_diffim? (Or, alternately, could the one in ip_diffim be called `useSubfilterMidpoint`? Mostly I'm seeking consistency!)

Reply:
I've switched the name to `splitSubfilters`, and used it consistently throughout.